core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -366,6 +366,12 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
+  test("add jar local path with comma") {
+    sc = new SparkContext(new SparkConf().setAppName("test").setMaster("local"))
```

Comment:
Cold you add tests for other other schemas?

Reply:
> Cold you add tests for other other schemas?

Updated